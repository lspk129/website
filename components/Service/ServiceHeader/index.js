import styled from 'styled-components';

const ServiceHeader = styled.h2`
  text-align: center;
  padding: 60px;
  margin: 0;
`;

export default ServiceHeader;
