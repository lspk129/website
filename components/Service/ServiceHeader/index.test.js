import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ServiceHeader from './';

describe('ServicesSection', () => {
  it('renders correctly', () => {
    const wrapper = renderer.create(<ServiceHeader />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders h2 element', () => {
    const wrapper = shallow(<ServiceHeader />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('h2');
  });
});
