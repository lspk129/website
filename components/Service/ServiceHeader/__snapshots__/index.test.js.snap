// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceHeader renders correctly 1`] = `
<h2
  className="sc-bdVaJa bHBKzw"
/>
`;
